model/qti/Element.php

```diff
@@ -282,23 +319,23 @@ public function getIdentifiedElement($identifier, $elementClasses = [])
 
     /**
      * Check if an attribute exists within the Qti Element
-     *
-     * @param string $name
-     * @return boolean
      */
-    public function hasAttribute($name)
+    public function hasAttribute(string $name): bool
     {
         return isset($this->attributes[$name]);
     }
 
     /**
      * Short handy method to get/set an attribute value
      *
-     * @param string $name
      * @param mixed $value
+     *
      * @return mixed
+     *
+     * @throws QtiModelException
+     * @throws AttributeException
      */
-    public function attr($name, $value = null)
+    public function attr(string $name, $value = null)
```

Comment:
This is nullable. It is better to double check the methods internals to see if the typehints will not break.

Reply:
It's not used without param anywhere in `usages` list